### DISCUSSION TITLE
Method to parse Example Value from  Request Body Object

### DISCUSSION THREAD

**Discussion by taivnn9** [swagger-api/swagger-ui#7184] [Q&A]
Hi, I would like to parse **Example Value (Json String)** from  [**Request Body(Java Object)**](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#requestBodyObject), 
Who did know how can I do it?
Thank in advance.

In detail:
**Input: Request Body Object** [(io.swagger.v3.oas.models.parameters.RequestBody)](http://docs.glngn.com/latest/api/io.swagger.core.v3.swagger-models/index.html?io/swagger/v3/oas/models/parameters/RequestBody.html)
**Target: Json String**
{
  "id": 10,
  "name": "doggie",
  "category": {
    "id": 1,
    "name": "Dogs"
  },
  "photoUrls": [
    "string"
  ],
  "tags": [
    {
      "id": 0,
      "name": "string"
    }
  ],
  "status": "available"
}

**Comment by henimbola** [comment]
Hi @taivnn9 , did you find the solution ?
